### PR TITLE
students.yml with naveen

### PR DESCRIPTION
**Pull request by naveenbalachandran**

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -444,3 +444,11 @@
   image: rahul.jpg
   twitter: 448bb8
   bio: Programmer
+
+- name: Naveen Balachandran
+  github: naveenbalachandran
+  image: naveen.jpg
+  twitter: thenaveenb
+  facebook: Naveen Balachandran
```
Comment (jigyasa-grover) [inline _data/students.yml:452]: Add your user id and not display name.

**Comment by jigyasa-grover** [comment]
Did you add your picture in `img/students` folder ?

**Comment by naveenbalachandran** [comment]
I couldnt but I did a pull request #468 with my photo. My internet
connection is poor so I couldnt post it

**Comment by jigyasa-grover** [comment]
It's still not added there. Also correct the `facebook:` entry.

**Comment by naveenbalachandran** [comment]
WhaT ?

**Comment by naveenbalachandran** [comment]
jid08 What is wrong with fb entry?

**Comment by jigyasa-grover** [comment]
I have added a note on https://github.com/fossasia/gci15.fossasia.org/pull/469#discussion-diff-49943195R452 . You need to add the user id and not the display name.

**Comment by naveenbalachandran** [comment]
ok, that I understand but I cant upload picture

**Comment by naveenbalachandran** [comment]
naveen
Could you please add this picture

**Comment by naveenbalachandran** [comment]
Pleaseeeeeee :+1:

**Comment by naveenbalachandran** [comment]
@jid08 Are you here

**Comment by jigyasa-grover** [comment]
I could easily do that but that shall defeat the purpose of the task !

I am re-posting my comment from your prev PR #468 

"_Well, you have claimed the corresponding GCI task @naveenbalachandran_

_You could start afresh by:_
- _Forking the repo_
- _Adding your photo in_ `img/students` 
- _Making an entry of your name an details in_ `students.yml` 
- _Sending a new Pull Request (PR) for the same :)_"

**Comment by naveenbalachandran** [comment]
Could you tell how to fix You dont have permission to modify this repo

**Comment by naveenbalachandran** [comment]
This is what happens

**Comment by jigyasa-grover** [comment]
Yes, because you need to push the changes to your fork and not this repo itself.

**Comment by naveenbalachandran** [comment]
ohh ok

**Comment by SaptakS** [comment]
Also, can you please close one of the PRs. I guess PR #469  and #468 are same..

**Comment by jigyasa-grover** [comment]
Closing this. A new PR #475 has been made in this regard.

**Pull request closed without merging**